docs/api/app.md

```diff
@@ -331,6 +331,10 @@ to the npm modules spec. You should usually also specify a `productName`
 field, which is your application's full capitalized name, and which will be
 preferred over `name` by Electron.
 
+### `app.setName()`
```

Comment:
Nice catch. Unified the format and squashed in https://github.com/atom/electron/pull/4851/commits/8cf4574198da33976ffc0e3c6d87d6aac6e6f48c